lib/app/layouts/settings/pages/server/server_management_panel.dart

```diff
@@ -77,20 +80,42 @@ class ServerManagementPanelController extends StatefulController {
       timeSync.value = response.data['data']['macos_time_sync'];
       hasCheckedStats.value = true;
 
-      http.serverStatTotals().then((response) {
-        if (response.data['status'] == 200) {
-          stats.addAll(response.data['data'] ?? {});
-          http.serverStatMedia().then((response) {
+      final subsequentRequests = <Future>[];
+
+      subsequentRequests.add(  
+        http.serverStatTotals().then((response) {
+          if (response.data['status'] == 200) {
+            stats.addAll(response.data['data'] ?? {});
+            http.serverStatMedia().then((response) {
+              if (response.data['status'] == 200) {
+                stats.addAll(response.data['data'] ?? {});
+              }
+            });
+          }
+        }).catchError((_) {
+          showSnackbar("Error", "Failed to load server statistics!");
+        })
+      );
+
+      if (privateAPIStatus.value) {
+        subsequentRequests.add(
+          http.getAccountInfo().then((response) {
             if (response.data['status'] == 200) {
-              stats.addAll(response.data['data'] ?? {});
+              accountAliases.clear();
+              final aliases = ((response.data['data']['vetted_aliases'] ?? []) as List<dynamic>).map((a) => a['Alias'] as String);
+              accountAliases.addAll(aliases);
+            
+              hasAccountInfo.value = true;
             }
-          });
-        }
-        opacity.value = 1.0;
-      }).catchError((_) {
-        showSnackbar("Error", "Failed to load server statistics!");
-        opacity.value = 1.0;
-      });
+          })
+          .catchError((e) { Logger.debug("Failed to get account info $e"); })
+        );
+      }
+
+      Future.wait(subsequentRequests)
+        .then((value) {})
+        .catchError((_) { })
```

Comment:
Are these necessary? If we catchError here, not sure that it will get propagated to line 120.

Reply:
Will remove